frontend/src/app/interfaces/basic-info.ts

```diff
@@ -0,0 +1,15 @@
+/*
```

Comment:
The name of the folder should be "model". And the file should also end on ".model.ts".

I suggest: You should move the file to app/forms/model/basic-info.model.ts or to app/forms/steps/basic-info/model/basic-info.model.ts.

Reply:
Ya I will change it by tomorrow.